NEWS.md

```diff
@@ -1,9 +1,10 @@
-# EpiNow2 1.4.9000
+# EpiNow2 1.4.9001
```

Comment:
we have stopped doing this (and if this isn't flagged in the contributing guide (or rather not flagged) it should/shouldn't be.

Reply:
Oh, really? Not sure I understand. Do you mean incrementing by 1 or incrementing in general? What should/shouldn't be flagged?

Reply:
Broadly following the [Tidyverse package versioning](https://r-pkgs.org/lifecycle.html) (which perhaps isn't mentioned in the contribution guide)

> Increment the development version, e.g. from 9000 to 9001, if you’ve added an important feature and you (or others) need to be able to detect or require the presence of this feature. For example, this can happen when two packages are developing in tandem. This is generally the only reason that we bother to increment the development version. This makes in-development versions special and, in some sense, degenerate. Since we don’t increment the development component with each Git commit, the same package version number is associated with many different states of the package source, in between releases.

Reply:
I think the flagging was referring to the whether this is mentioned in the contribution guide or not

Reply:
Thanks, I'll remove the increment. Are we going to determine on a case-by-case basis when to do the increment? Based on the snippet above, it seems something like changes to a vignette or the addition of one shouldn't warrant an increment. The guide already mentions that we ask for a NEWS item to be added.

Reply:
Yes I think case-by-case with a strong default to no and only in exceptions if there is a specific need (which I currently can't imagine what it would be).

Reply:
I've made the change here 517968b5fef7805826cd39f6422e63d1733a9b34.